.ruby-version

```diff
@@ -1 +1 @@
-jruby-9.1.12.0
+jruby-9.3.4.0
```

Comment:
This brings `.ruby-version` into alignment with the `versions.yml` file, for all who use rbenv